src/Services/CustomerOptionValueArchiver.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brille24\SyliusCustomerOptionsPlugin\Services;
+
+use Brille24\SyliusCustomerOptionsPlugin\Entity\OrderItemInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Order\Model\OrderInterface;
+use Webmozart\Assert\Assert;
+
+final class CustomerOptionValueArchiver implements OrderItemValueCopierInterface
```

Comment:
This cannot be an Archiver, since a future issue will describe how to actually archive customer options analog to for example shipping methods.